sku.go

```diff
@@ -572,3 +572,13 @@ func (s *SKU) Equal(other *SKU) bool {
 		localErr != nil &&
 		otherErr != nil
 }
+
+// Includes returns true if the SKU's name is in the list of SKUs.
+func (s *SKU) Includes(skuList []SKU) bool {
```

Comment:
`IsIncludedIn` may better reflect the use? In any case, this does not have a signature of `FilterFn` which is expected in `List`, etc; may need a filter builder like one of these? 
https://github.com/Azure/skewer/blob/6f2365e468f11ca4f3d065eb770a8b6512d68fba/cache.go#L335-L363

Reply:
yeah, ran out of time on Friday, added the actual FilterFn, see the new `IncludesFilter`